docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -184,10 +185,14 @@ where the bare-metal machines will be attached.
           oc create -v baremetalMachineSet.yaml
 
 
+## Known issues
+
+Bare metal nodes are not supported on clusters that use Kuryr.
```

Comment:
is it not possible to add a validation that prevents users from using BM instance types when Kuryr is enabled?

Reply:
Unfortunately it is not obvious how to implement that validation, as the Installer is not aware what Nova flavors are BMs.

Reply:
It might be possible to know if we're deploying on BM by getting all BM nodes when ironic service is enabled, and checking whether they map to the related flavors, but definitely not trivial.